src/jobservice/period/mock_scheduler.go

```diff
@@ -3,7 +3,6 @@
 package period
 
 import mock "github.com/stretchr/testify/mock"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380